survey_goals_and_questions.md

```diff
@@ -0,0 +1,169 @@
+The draft survey itself.
+
+# Survey length
+Based on feedback from the community, a ~10 minute survey is ideal and would maximize participation likelihood.  Beyond length, we will make most questions optional so that participants can pick and choose questions if they are rushed or prefer not to answer.  Progress will be shown throughout.
+
+# Survey Topics
+Based on discussions with many in the community, the below topics were most frequently mentioned as items folks hoped to learn about the community / were perceived as most valuable to learn about the community.
+
+Each topic has:
+* Estimated timing
+* What we want to learn
+* Outcomes we hope to achieve by learning
+* List of questions to be asked
+
+# Who we are: 5 mins
+## [2 mins] Basic demographics of person & company 
+### What do we learn / get by asking this?
+* Get a baseline view of the types of orgs that are applying MLOps
+* Get a baseline view of how ML teams are structured & who they work with
+* Use as filters to analyze other questions
+### Questions
+* What is your current job title?
+* What bucket does your role fall in?
+    * ML Engineer
+    * Data Scientist
+    * Software Engineer
+    * Data Engineer
+    * SRE/DevOps
+    * Researcher
+    * Data Analyst
+* What industry do you work in?
+* What type of company?
+    * Startup
+    * Big tech
+    * Enterprise [need better name]
+* How many FTE in your company (number of people)?
+* How many FTE on your team?
+* How many FTE working on data science & machine learning?
+* Where in your organization does your team sit?
+    * Within central services
+    * Part of a business unit
+    * other??  how do we make this question relevant to startup & big E?
+* Which other roles do you collaborate with on MLOps?
```

Comment:
compliance, secuity, board of directors, 

maybe another question could be who do you report to directly. how many direct reports do you have

Reply:
```suggestion
* Which other roles do you collaborate with on MLOps?
    * Other (please describe)
    * Compliance
    * Security
    * Business stakeholders (executives/directors)
    * Business stakeholders (managers)
```